UPGRADING.md

```diff
@@ -1,218 +1,144 @@
-
-##### Expected upgrade result
-
-When the upgrade block height is reached, Gaia will panic and stop:
-
-This may take a few minutes to a few hours.
-After upgrade, the chain will continue to produce blocks when validators with a total sum voting power > 2/3 complete their node upgrades.
-
-#### Auto-Downloading the Gaia binary
-
-**This method is not recommended!**
-
-## Upgrade duration
-
-The upgrade may take a few minutes to several hours to complete because cosmoshub-4 participants operate globally with differing operating hours and it may take some time for operators to upgrade their binaries and connect to the network.
-
-## Rollback plan
-
-During the network upgrade, core Cosmos teams will be keeping an ever vigilant eye and communicating with operators on the status of their upgrades. During this time, the core teams will listen to operator needs to determine if the upgrade is experiencing unintended challenges. In the event of unexpected challenges, the core teams, after conferring with operators and attaining social consensus, may choose to declare that the upgrade will be skipped.
-
-Steps to skip this upgrade proposal are simply to resume the cosmoshub-4 network with the (downgraded) v14.2.0 binary using the following command:
-
-```shell
-gaiad start --unsafe-skip-upgrade <UPGRADE_HEIGHT>
+Restart=On-failure 
 ```
+Reload the service control `sudo systemctl daemon-reload`.
 
-Note: There is no particular need to restore a state snapshot prior to the upgrade height, unless specifically directed by core Cosmos teams.
-
-Important: A social consensus decision to skip the upgrade will be based solely on technical merits, thereby respecting and maintaining the decentralized governance process of the upgrade proposal's successful YES vote.
-
-## Communications
-
-Operators are encouraged to join the `#cosmos-hub-validators-verified` channel of the Cosmos Hub Community Discord. This channel is the primary communication tool for operators to ask questions, report upgrade status, report technical issues, and to build social consensus should the need arise. This channel is restricted to known operators and requires verification beforehand. Requests to join the `#cosmos-hub-validators-verified` channel can be sent to the `#general-support` channel.
-
-## Risks
-
-As a validator performing the upgrade procedure on your consensus nodes carries a heightened risk of double-signing and being slashed. The most important piece of this procedure is verifying your software version and genesis file hash before starting your validator and signing.
-
-The riskiest thing a validator can do is discover that they made a mistake and repeat the upgrade procedure again during the network startup. If you discover a mistake in the process, the best thing to do is wait for the network to start before correcting it.
-
-## Reference
+# Revert `gaiad` configurations
```

Comment:
shouldn't that step be put somewhere before starting the new gaiad version?

Reply:
Moved.